templates/github/.ci/ansible/settings.py.j2.copy

```diff
@@ -20,6 +20,12 @@ API_ROOT = {{ api_root | repr }}
 {% endfor %}
 {% endif %}
 
+{% if pulp_scenario_settings is defined and pulp_scenario_settings %}
+{% for key, value in pulp_scenario_settings.items() %}
+{{ key | upper }} = {{ value | repr }}
+{% endfor %}
+{% endif %}
+
```

Comment:
why not just use `pulp_settings`?

Reply:
Because i want different values and different settings in different scenarios. That way we can test the feature being on and off.

Reply:
only now I realize it is only applied to the S3 job,
I think the name is misleading, s6 sounds like a specific single container config

Reply:
I'll add the same logic for the other scenarios then.